README.rst

```diff
@@ -137,6 +137,15 @@ or:
 .. code-block:: bash
 
    brew cask install iterative/homebrew-dvc/dvc
+   
+Conda
```

Comment:
Let's fix also a sentence above: "There are three options ..." .. also I'm not sure but it might make sense to use Anaconda? just for search engines